Week1/homework/js2-exercises/1-books.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Book list</title>
+</head>
+
+<body>
+  <h1>My book list</h1>
+
+  <script>
+    const books = [
+    {
+    title : 'An Artist of the Floating World',
+    author : 'Kazuo Ishiguro',
+    alreadyRead :false  
+    },
+    {
+    title : 'Underworld',
+    author : 'Don DeLillo',
+    alreadyRead :true   
+    }
+];
+
+    const ul = document.createElement('ul');
+    // 1
+    for (let obj of books) {
+// 2
+      const p = document.createElement('p');
+      p.innerText = `${obj.title} was written by ${obj.author.toUpperCase()}`;
+// 5
+      
+      if (obj.alreadyRead === true) {
+        p.style.color = 'green';
+      } else {
+        p.style.color = 'red';
+      }
+    
+// 3
+      const li = document.createElement('li');
+      li.appendChild(p);
+      ul.appendChild(li);
+      document.body.appendChild(ul);
+    }
+    // 4
+    let img1 = document.createElement('img');
+    let img2 = document.createElement('img');
```

Comment:
why did you have img outside the for of loop. Think about how is img1 different than img2 and how can you solve that. In the end you need an image for each book right?